src/EventListener/Contao/LoadDataContainerListener.php

```diff
@@ -91,4 +97,17 @@ protected function prepareReaderConfigElementTable(string $table)
 
         $GLOBALS['TL_LANG'][$table]['config_element_config_legend'] = $this->translator->trans('huh.syndication_type.legends.tl_reader_config_element.config_element_config_legend');
     }
+
+    protected function prepareContentTable(string $table)
+    {
+        if (isset($this->bundleConfig['enable_content_syndication']) && true === $this->bundleConfig['enable_content_syndication']) {
+            $dca = &$GLOBALS['TL_DCA']['tl_content'];
+            $dca['palettes'][SyndicationElement::TYPE] = '{type_legend},type;{template_legend:hide},customTpl;{syndication_config_legend},titleText,text;'.$this->syndicationTypeDcaProvider->getPalette(true);
+            $dca['fields']['text']['label'] = $GLOBALS['TL_LANG']['tl_page']['description'];
```

Comment:
Du überschreibst die Werte hier global (also für alle Content-Elemente). Element-abhängige DCA-Anpassungen müssen in ein onload-Callback.